src/StoreApi/Utilities/ProductQuery.php

```diff
@@ -40,6 +40,8 @@ public function prepare_objects_query( $request ) {
 		if ( ! empty( $request['sku'] ) ) {
 			$args['post_type'] = [ 'product', 'product_variation' ];
 		}
+		// Taxonomy query to filter products by type, category, tag, shipping class, and attribute.
```

Comment:
Removed the whitespace but also removed the line break in the process. Adding the line break back in.

```suggestion

		// Taxonomy query to filter products by type, category, tag, shipping class, and attribute.
```